src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -36,6 +62,10 @@ def load_arguments(self, _):
         c.argument('file_system', help='The file system of the data lake storage account.')
         c.argument('sql_admin_login_user', options_list=['--sql-admin-login-user', '-u'],
                    help='The sql administrator login user name.')
+        c.argument('enable_managed_virtual_network', options_list=['--enable-managed-vnet',
+                                                                   '--enable-managed-virtual-network'],
+                   arg_type=get_three_state_flag(),
+                   help='The flag indicates whether enable managed virtual network.')
```

Comment:
>help='The flag indicates whether enable managed virtual network.') [](start = 19, length = 66)

Better to add the description of the default value

Reply:
The arg tye is `get_three_state_flag` when we run `az synapse workspace create -h`, for this parameter the help message is as bellow:
The flag indicates whether enable managed virtual network.  Allowed values: false, true. 
I think it is clear enough.

---
In reply to: [533221222](https://github.com/Azure/azure-cli/pull/16095#discussion_r533221222) [](ancestors = 533221222)